lib/global-styles.php

```diff
@@ -262,26 +262,50 @@ function gutenberg_experimental_global_styles_get_theme() {
 }
 
 /**
- * Returns the style features a particular block supports.
+ * Return how the style property is structured.
  *
- * @param array $supports The block supports array.
+ * @return array Style property structure.
+ */
+function gutenberg_experimental_global_styles_get_style_property() {
+	return array(
+		'line-height'              => array( 'typography', 'lineHeight' ),
+		'font-size'                => array( 'typography', 'fontSize' ),
+		'background'               => array( 'color', 'gradient' ),
+		'background-color'         => array( 'color', 'background' ),
+		'color'                    => array( 'color', 'text' ),
+		'--wp--style--color--link' => array( 'color', 'link' ),
+	);
+}
+
+/**
+ * Return how the support keys are structured.
  *
- * @return array Style features supported by the block.
+ * @return array Support keys structure.
  */
-function gutenberg_experimental_global_styles_get_supported_styles( $supports ) {
-	$style_features = array(
+function gutenberg_experimental_global_styles_get_support_keys() {
+	return array(
 		'--wp--style--color--link' => array( '__experimentalColor', 'linkColor' ),
-		'background-color'         => array( '__experimentalColor' ),
```

Comment:
These keys were renamed to use the same conventions as client-side (camelCase over kebab-case), making everything more coherent and less computation-demanding.